crates/edr_napi/dist/src/ts/solidity_tests.d.ts

```diff
@@ -0,0 +1,17 @@
+import { StandardTestKind, FuzzTestKind, InvariantTestKind } from "../../index";
+export declare enum SortOrder {
+    Ascending = 0,
+    Descending = 1
+}
+export interface GasUsageFilter {
+    minThreshold?: bigint;
+    maxThreshold?: bigint;
+}
+export declare function extractGasUsage(testResults: Array<{
+    name: string;
+    kind: StandardTestKind | FuzzTestKind | InvariantTestKind;
+}>, filter?: GasUsageFilter, ordering?: SortOrder): Array<{
+    name: string;
+    gas: bigint;
+}>;
+//# sourceMappingURL=solidity_tests.d.ts.map
```

Comment:
Out of curiosity, what does this line do?

Reply:
This tells tooling (runtime, LSP, etc.) that this file has been compiled/transpiled from another source file, and specifies how to convert positions in this file into positions in the source file. This sort of thing is used by stack traces or by the editor's "go to definition".